README.md

```diff
@@ -1,34 +1,26 @@
-# EIPs [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.
+# Ethereum Improvement Proposals (EIPs) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-A browsable version of all current and draft EIPs can be found on [the official EIP site](http://eips.ethereum.org/).
```

Comment:
Did you intend to remove the link to the website?  I personally have found it valuable on multiple occasions.

Reply:
Added back in c6c1acf76328f8632473c8bed1ef08e8f688d2ae

Reply:
I believe this one is resolved